### PR TITLE
Added privatelaw config

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-839

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/bulk-scan-processor/values.yaml [hunk 1]
```diff
@@ -97,6 +98,10 @@ java:
           alias: idam.users.publiclaw.username
         - name: idam-users-publiclaw-password
           alias: idam.users.publiclaw.password
+        - name: idam-users-privatelaw-username
+          alias: idam.users.privatelaw.username
```
Comment (mdayican) [inline charts/bulk-scan-processor/values.yaml:102]: we do not have the secrets yet

File: charts/bulk-scan-processor/values.yaml [hunk 2]
```diff
@@ -34,6 +34,8 @@ java:
     DELETE_REJECTED_FILES_CRON: "0 0/10 * * * *"
     INCOMPLETE_ENVELOPES_TASK_CRON: "0 */15 * * * *"
     DELETE_REJECTED_FILES_TTL: PT72H
+    PUBLICLAW_ENABLED: "true"
```
Comment (mdayican) [inline charts/bulk-scan-processor/values.yaml:37]: no need this. 
default enabled true

File: charts/bulk-scan-processor/values.yaml [hunk 3]
```diff
@@ -34,6 +34,8 @@ java:
...
+    PRIVATELAW_ENABLED: "true"
```
Comment (mdayican) [inline charts/bulk-scan-processor/values.yaml:38]: should be false. 
for demo it should be true.

File: infrastructure/main.tf [hunk 4]
```diff
@@ -129,6 +129,8 @@ module "bulk-scan" {
     OCR_VALIDATION_URL_PROBATE             = "${var.ocr_validation_url_probate}"
 
     NO_NEW_ENVELOPES_TASK_ENABLED     = "false"
+    PUBLICLAW_ENABLED                 = "true"
+    PRIVATELAW_ENABLED                = "false"
```
[the remark this answers is not in the document]
Comment (apereverzin) [reply infrastructure/main.tf:133]: I have split this PR into https://github.com/hmcts/bulk-scan-processor/pull/2248, https://github.com/hmcts/bulk-scan-processor/pull/2249

**Pull request closed without merging**